### DISCUSSION TITLE
Not show the region name in the Meeting location address field

### DISCUSSION THREAD

**Discussion by came2believe** [code4recovery/12-step-meeting-list#1514] [Q&A]
Hi there, is there a way to not show the region name in the meeting location address field?

In the screenshot, marked in yellow is the name of the "region" = "Zurich"

The name of the location is "Crammerstrasse"

And the full address is also shown.

![Screenshot 2024-09-13 093604](https://github.com/user-attachments/assets/8e4984a8-a661-420f-8e31-4bd70f8800a0)

**Comment by tim-rohrer** [comment]
So you don't want to show "Zurich"? I suspect custom CSS would work.
